### PR TITLE
Added GlPortal.

### PR DESCRIPTION
**Pull request by hhirsch**

### CODE REVIEW

File: _data/projects/GlPortal.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+- c++
+- python
+- javascript
+- Game
+- engine
+- 3d
+upforgrabs:
+  name: up-for-grabs
+  link: https://bugs.glportal.de/
```
Comment (shiftkey) [inline _data/projects/GlPortal.yml]: It'd be great if this URL pointed to a filtered list of issues which new contributors might be able to get started on. Is this possible?

**Comment by hhirsch** [comment]
I have read the instructions. We have issues tagged up-for-grabs, but our bugtracker does not support filtering for them. I am willing to create a seperate list if it is necessary. Let me know what you think.

**Comment by shiftkey** [comment]
@hhirsch that URL is where the users will land after clicking on the project link, so if it's possible to make it less intimidating then I'd love to see that happen...

**Comment by hhirsch** [comment]
@shiftkey So flyspray can't filter by tag. But I found another way to filter for issues. I have created a new task type called "Simple Task". Now people will find a simple small list.

**Comment by hhirsch** [comment]
@shiftkey :white_check_mark: **It should meet your specs now.** You'll find specially selected issues under the url now. I will also add a bunch more so that there are always about 7 tasks from very easy to easy.

File: _data/projects/GlPortal.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+- c++
+- python
+- javascript
+- Game
+- engine
+- 3d
+upforgrabs:
+  name: Simple Task
+  link: https://bugs.glportal.de/index.php?string=&project=1&type%5B%5D=5
```
Comment (shiftkey) [inline _data/projects/GlPortal.yml]: You can drop the `string=&` parameter from here for a bit of extra :gem:

**Comment by shiftkey** [comment]
@hhirsch so good! :sparkles: just one little suggestion!

**Comment by hhirsch** [comment]
@shiftkey Hello. Shortened the URL.

**Comment by shiftkey** [comment]
Thanks!

**Pull request merged**